Adds instructions to deploy locally without Cloud Build. (#105)
amandakarina/terraform-example-foundation: 3-networks-dual-svpc/README.md

### Run Terraform locally

1. The next instructions assume that you are at the same level of the `terraform-example-foundation` folder. Create and change into `gcp-network` folder, copy `3-networks-dual-svpc` content, the Terraform wrapper script and ensure it can be executed. Also, initialize git so you can manage versions locally.

   ```bash
   mkdir gcp-network
   cp -R terraform-example-foundation/3-networks-dual-svpc/* gcp-network
   cp terraform-example-foundation/build/tf-wrapper.sh gcp-network/
   cp terraform-example-foundation/.gitignore gcp-network
   chmod 755 ./gcp-environments/tf-wrapper.sh
   ```

> 1. Navigate to `gcp-network` and initialize a local git repository, so you can manage versions locally. Create the environment branches.

Reviewer: ```suggestion
1. Navigate to `gcp-network` and initialize a local Git repository to manage versions locally. Then, create the environment branches.
```
